examples/using-guidebot-flags.md

```diff
@@ -0,0 +1,103 @@
+# Using Switch Cases with Guidebot
+
+This page is an explanation of how to use switch cases with a feature in Guidebot, `message.flags`.
+
+## An Example
+
+Switch cases are a nice feature that can be used without the `message.flags` feature, and can be used like this. 
+
+```js
+// This will look for the first argument. If your command was 'hi', and you did 'hi send', it would send 'Hi!' to the channel.
+const trigger = args[0];
+
+switch (trigger) {  
+  case ('send'): {
+    // This will send in the channel that the command was run in.
+    message.channel.send('Hi!');
+    break;
+  }
+
+  case ('dm'): {
+    // This will DM the message author..
+    message.author.send('Hi!');
+  }
+}
+```
+
+However, you can use the switch case with the neat little `message.flags` concept.
+
+```js
+  // This will look for the first argument beginning with -, the flag. If your command was 'hi', and you did 'hi -send', it would send 'Hi!' to the channel.
+  switch (message.flags[0]) {
+    case ('send'): {
+    // This will send in the channel that the command was run in.
+    message.channel.send('Hi!');
+    break;
+  }
+
+  case ('dm'): {
+    // This will DM the message author.
+    message.author.send('Hi!');
+  }
+}
+```
+
+## Using with Arguments
+
+Let's say you're trying to make a command to add or take a role to/from a user. This can be consolidated into one command using the `message.flags` method. This command is based off of the [Guidebot](https://github.com/AnIdiotsGuide/guidebot) framework, and is called role.
```

Comment:
Please refer [method](https://en.wikipedia.org/wiki/Method_(computer_programming)) as a function or procedure instead of a way or pattern of coding. `message.flags` is more like an extended property of Message. Calling it a method implies `message.flags()` and in my opinion, it's confusing.

Reply:
Changed the wording around.